src/components/list/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {FC} from 'react';
+import {Crypto, Props} from '../interfaces';
+import {
+  TextList,
+  Container,
+  CryptoLogo,
+  Line,
+  Arrow,
+  TextPercent,
+  TextSymbol,
+} from './styles';
+import ArrowUp from '../../images/icons/ArrowUp.png';
+import ArrowDown from '../../images/icons/ArrowDown.png';
+
+const List: FC<Props> = ({item}: {item: Crypto}) => (
```

Comment:
You should avoid the usage of this FC because of the following:

- It will be deprecated in the next version of React (major reason)
- You are using typed props or interfaces, so that should be enough.